### ISSUE TITLE
Style code examples

### ISSUE THREAD

**Issue by teenwolfblitzer** [Kajabi/sage-archived#30]
[Code prettifier](https://github.com/google/code-prettify) is active in the docs to allow styling/theming of code snippets.

- Update styling with the following requirements:
  - Colors should be within Sage palette
  - Text maintains WCAG level AA compliance
